src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ReferenceBlockConcordanceIntegrationTest.java

```diff
@@ -0,0 +1,436 @@
+                                new TestReferenceBlockConcordanceVariant("C", 10, 10, 99)
+                        ),
+                        // Truth block histogram
+                        Stream.of(new Object[][] {
+                                { "10,99", 1}
+                        }).collect(Collectors.toMap(data -> (String) data[0], data -> (Integer) data[1])),
+                        // Eval block histogram
+                        Stream.of(new Object[][] {
+                                { "7,99", 1}
+                        }).collect(Collectors.toMap(data -> (String) data[0], data -> (Integer) data[1])),
+                        // Confidence concordance
+                        Stream.of(new Object[][] {
+                                { "99,99", 7}
+                        }).collect(Collectors.toMap(data -> (String) data[0], data -> (Integer) data[1])),
+                },
+        };
+    }
+
+    @Test(dataProvider = "provideSyntheticGVCFs")
+    public void testSyntheticGVCFs(final List<TestReferenceBlockConcordanceVariant> truthVariants, final List<TestReferenceBlockConcordanceVariant> evalVariants, final Map<String, Integer> expectedTruthBlockHistogram, final Map<String, Integer> expectedEvalBlockHistogram, final Map<String, Integer> expectedConfidenceConcordance) throws Exception {
+        Pair<File, File> inputFiles = writeTestGVCFs(truthVariants, evalVariants);
+
+        final Path truthBlockHistogramFile = createTempPath("truth_block_histogram", ".tsv");
+        final Path evalBlockHistogramFile = createTempPath("eval_block_histogram", ".tsv");
+        final Path confidenceConcordanceHistogramFile = createTempPath("confidence_concordance_histogram", ".tsv");
+
+        final String[] args = {
+                "--" + AbstractConcordanceWalker.TRUTH_VARIANTS_LONG_NAME, inputFiles.getLeft().toString(),
+                "--" + AbstractConcordanceWalker.EVAL_VARIANTS_LONG_NAME, inputFiles.getRight().toString(),
+                "--" + ReferenceBlockConcordance.TRUTH_BLOCK_HISTOGRAM_LONG_NAME, truthBlockHistogramFile.toString(),
+                "--" + ReferenceBlockConcordance.EVAL_BLOCK_HISTOGRAM_LONG_NAME, evalBlockHistogramFile.toString(),
+                "--" + ReferenceBlockConcordance.CONFIDENCE_CONCORDANCE_HISTOGRAM_LONG_NAME, confidenceConcordanceHistogramFile.toString(),
+        };
+        runCommandLine(args);
+
+        MetricsFile<?, String> truthBlockMetrics = new MetricsFile<>();
+        truthBlockMetrics.read(new FileReader(truthBlockHistogramFile.toFile()));
+        if (truthBlockMetrics.getNumHistograms() > 0) {
+            Assert.assertEquals(truthBlockMetrics.getNumHistograms(), 1);
+            Histogram<String> truthBlockHistogram = truthBlockMetrics.getHistogram();
+            // Check value and remove entry from expected histogram...
+            truthBlockHistogram.values().forEach(bin -> {
```

Comment:
if you make the expected histograms actual `Histogram` objects you can just use the `Histogram::equals` method.